cli/checkpoint/checkpoint.go

```diff
@@ -0,0 +1,40 @@
+package checkpoint
+
+import (
+	"bytes"
+	"fmt"
+	"os/exec"
+
+	"github.com/tarantool/tt/cli/cmdcontext"
+)
+
+// CmdOpts contains flags for managing checkpoint files commands.
+// Used for commands tt cat and tt play, which are checkpoint files commands.
+type CmdOpts struct {
+	To         uint64
+	From       uint64
+	Space      []int
+	Format     string
+	Replica    []int
+	ShowSystem bool
+}
+
+// Cat returns the contents of .snap/.xlog files.
+func Cat(cmdCtx *cmdcontext.CmdCtx) (stdout []byte, err error) {
+	var outbuff, errbuff bytes.Buffer
+	cmd := exec.Command(cmdCtx.Cli.TarantoolExecutable, "-")
+	cmd.Stdout, cmd.Stderr = &outbuff, &errbuff
+
+	StdinPipe, err := cmd.StdinPipe()
+	if err != nil {
+		return nil, err
+	}
+	StdinPipe.Write([]byte(catFile))
+	StdinPipe.Close()
+
+	if err := cmd.Run(); err != nil {
+		return outbuff.Bytes(), fmt.Errorf("Result of cat: %s", errbuff.String())
```

Comment:
`return nil, fmt.Errorf(...)`

Reply:
Similar to [this](https://github.com/tarantool/tt/pull/48#discussion_r870217103)

But here we output the files that were processed before the error occurred